data/brands/shop/clothes.json

```diff
@@ -5362,6 +5364,18 @@
         "shop": "clothes"
       }
     },
+    {
+      "displayName": "NAME IT",
+      "id": "nameit-3937bd",
+      "locationSet": {"include": ["001"]},
```

Comment:
001 means this brand is suitable for every countries and region, it will be a global entry. For Japan, Africa, China, and many other places. If this brand only appear in Germany, you can use `de`, or if you can see it in many Europe countries, `eu` is also acceptable.

This comment also apply to other 001 entry added in this PR.

Reply:
applied a fix